### DISCUSSION TITLE
what's the difference with mozilla/DeepSpeech

### DISCUSSION THREAD

**Discussion by zh794390558** [coqui-ai/STT#1812] [Q&A] [answered]
what's the difference with mozilla/DeepSpeech

**Comment by reuben** [comment, marked as answer]
You can read more about Coqui on our [website](https://coqui.ai/about) and the Discourse [post](https://discourse.mozilla.org/t/a-new-in-speech-tech-town/77026/4) we made announcing the launch. The main difference is what it says on the tin: Coqui is a different organization, founded by the main authors of the projects and focused entirely on speech tech.
